### PR TITLE
Add instructions for replacing fedora-flatpak remote with flathub

### PR DESCRIPTION
**Pull request by DitherWither**
I had made a similar pull request a few minutes ago but decided close it as it needed a few edits

I have no idea how to edit a pull request

### CODE REVIEW

**Comment by travier** [comment]
You can push more commits on top of the branch to add them to the PR.

File: modules/ROOT/pages/tips-and-tricks.adoc [hunk 1]
```diff
@@ -160,3 +160,21 @@ or setup shell `alias`es as needed to make them available to the CLI like so:
   $ alias evince="flatpak run org.gnome.Evince"
     # or alias evince="org.gnome.Evince"
   $ evince
+
+== How do to replace fedora-flatpak remote with flathub?
```
Comment (travier) [inline modules/ROOT/pages/tips-and-tricks.adoc:164]: We usually add an empty line after the title

File: modules/ROOT/pages/tips-and-tricks.adoc [hunk 2]
```diff
@@ -160,3 +160,21 @@ or setup shell `alias`es as needed to make them available to the CLI like so:
...
+. Enable the flathub remote if not already enabled using the instructions {docs-url}/getting-started/#flathub-setup[here]
```
Comment (travier) [inline modules/ROOT/pages/tips-and-tricks.adoc:165]: How does that interact with the filtered flathub view available by default? Maybe we should make sure to remove this one before?

**Comment by kalev** [comment]
I am not sure it makes sense to have docs on a Fedora site how to replace Fedora provided software with a third party. What's the goal with this?

File: modules/ROOT/pages/tips-and-tricks.adoc (hunk 2 above)
Comment (travier) [reply modules/ROOT/pages/tips-and-tricks.adoc:166]: Yes, updating the getting started guide to get the unfiltered Flathub is probably better.

**Comment by travier** [comment]
We have the Flathub remote available via the third party repositories now so we probably don't need that anymore.

**Pull request closed without merging**